linkup/src/session.rs

```diff
@@ -315,7 +317,7 @@ pub fn create_preview_req_from_json(input_json: String) -> Result<Session, Confi
         Err(e) => Err(ConfigError::JsonFormat(e)),
         Ok(c) => {
             let server_conf = StorableSession {
-                session_token: String::from("potato"),
+                session_token: String::from(PREVIEW_SESSION_TOKEN),
```

Comment:
**_thought/suggestion (non-blocking):_**
I think this will allocate the string on the heap regardless, and since the `PREVIEW_SESSION_TOKEN` is only used here, I think you are fine to just put that string directly here.
Unless you are doing this for readability and not just to have a const. Then I would say is fine.

P.s. Love the fact that "potato" lived this long 😆

Reply:
Yeah mostly just for readability, not too worried about performance 😅